src/Service/Parsers/ODataASTVisitor.cs

```diff
@@ -79,13 +90,13 @@ public override string Visit(SingleValuePropertyAccessNode nodeIn)
         /// <returns>String representing param that holds given value.</returns>
         public override string Visit(ConstantNode nodeIn)
         {
-            if (nodeIn.TypeReference is null)
+            if (nodeIn.TypeReference is not null)
```

Comment:
This change is semantically the same as before, any reason why you had to do it?

Reply:
Nope its the same. Already had it reverted in my uncommitted changes. Done now.

Reply:
This doesn't look to have been reverted?